code/modules/atmospherics/machinery/pipes/layermanifold.dm

```diff
@@ -77,18 +73,18 @@
 /obj/machinery/atmospherics/pipe/layer_manifold/proc/findAllConnections()
 	front_nodes = list()
 	back_nodes = list()
-	var/list/new_nodes = list()
+	nodes = list()
```

Comment:
I assume the whole type wasn't avoiding using nodes variable for any particular reason ?

Reply:
afaik it wasn't using nodes, because it used it's own front_nodes and back_nodes variables. but that breaks the contract the superclass